src/vega.schema.ts

```diff
@@ -503,187 +525,187 @@ export type VgBinding = VgCheckboxBinding | VgRadioBinding |
  */
 export interface VgAxisBase {
   /**
-   * A boolean flag indicating if the domain (the axis baseline) should be included as part of the axis.
-   *
-   * __Default value:__ `true`
-   */
-  domain?: boolean;
-
-  /**
-   * A boolean flag indicating if grid lines should be included as part of the axis
-   *
-   * __Default value:__ `true` for [continuous scales](https://vega.github.io/vega-lite/docs/scale.html#continuous) that are not binned; otherwise, `false`.
+   * An interpolation fraction indicating where, for `band` scales, axis ticks should be positioned. A value of `0` places ticks at the left edge of their bands. A value of `0.5` places ticks in the middle of their bands.
```

Comment:
0.5 is the default according to https://github.com/vega/vega-parser/blob/master/src/config.js.